packages/snippet-bot/src/snippet-bot.ts

```diff
@@ -12,28 +12,127 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import {Application} from 'probot';
+import {Application, Octokit} from 'probot';
+import {parseRegionTags} from './region-tag-parser';
 import {logger} from 'gcf-utils';
+import * as minimatch from 'minimatch';
+
+type Conclusion =
+  | 'success'
+  | 'failure'
+  | 'neutral'
+  | 'cancelled'
+  | 'timed_out'
+  | 'action_required'
+  | undefined;
+
+interface ConfigurationOptions {
+  ignoreFiles: string[];
+}
 
 const CONFIGURATION_FILE_PATH = 'snippet-bot.yml';
 
-interface Configuration {
-  randomBoolean: boolean;
+class Configuration {
+  private options: ConfigurationOptions;
+  private minimatches: minimatch.IMinimatch[];
+
+  constructor(options: ConfigurationOptions) {
+    this.options = options;
+    this.minimatches = (options as ConfigurationOptions).ignoreFiles.map(
```

Comment:
I don't think you need the cast here `as ConfigurationOptions`

Reply:
done